### DISCUSSION TITLE
DB Integration with Apache Ignite

### DISCUSSION THREAD

**Discussion by doreilly257** [Budibase/budibase#3568] [Ideas]
It would be great to replace Redis/Couchbase with Apache Ignite for caching, in-memory and disk persistence.

**Comment by mjashanks** [comment]
Hey @doreilly257 

Thanks for the suggestion. We won't be adding an integration to Apache Ignite soon, but happy to accept contributions for it.

You mentioned "It would be great to replace Redis/Couchbase..." - are you suggesting that we replace `Redis` and `CouchDB` from our core stack? That won't be happening as it's such a fundamental change. Maybe I misunderstood you here.
